src/cljs/main/broadfcui/page/workspace/monitor/workflow_details.cljs

```diff
@@ -173,9 +189,9 @@
      (let [wlogurl (str "gs://" bucketName "/" submission-id "/workflow.logs/workflow."
                         (workflow "id") ".log")]
        (create-field "Workflow Log" (display-value wlogurl (str "workflow." (workflow "id") ".log"))))]
-    [WorkflowTiming {:label "Workflow Timing" :data raw-data :workflow-name workflow-name}]]
-
-   [:div {:style {:marginTop "1em" :fontWeight 500}} "Calls:"]
+    (when-not (core/empty? (workflow "calls")) [WorkflowTiming {:label "Workflow Timing" :data raw-data :workflow-name workflow-name}])
```

Comment:
You shouldn't need to import core, and for `when`s that are this long I'd like to see the "then" on the next line.